source/reference/minio-mc.rst

```diff
@@ -617,13 +625,24 @@ You can also define some of these options using :ref:`Environment Variables <min
 
    Alternatively, set the environment variable :envvar:`MC_QUIET`.
 
-.. option:: --insecure
+.. option:: --resolve
```

Comment:
Does this need versionadded? It's new I think. (Some other items on this page were only rearranged.)

Reply:
Added!